Framework/DataHandling/inc/MantidDataHandling/SaveAscii2.h

```diff
@@ -90,6 +90,7 @@ class DLLExport SaveAscii2 : public API::Algorithm {
   std::vector<std::string> m_metaData;
   std::map<std::string, std::vector<std::string>> m_metaDataMap;
   std::unique_ptr<AxisHelper::AxisProxy> m_axisProxy;
+  static const std::vector<std::string> ASCII_EXTS;
```

Comment:
if it is going to be a member property of SaveAscii2 keep an `m_asciiExts` form. or if you keep it as `ASCII_EXTS` have it as a global in the cpp file.

Reply:
I change this to a member of the class named `m_asciiExts`